doc/modules/language-guide/pages/message-inspection.adoc

```diff
@@ -0,0 +1,132 @@
+= Message Inspection
+:proglang: Motoko
+:company-id: DFINITY
+
+
+On the {IC}, a canister can selectively
+_inspect_ and _accept_ or _decline_ ingress messages submitted through the HTTP interface:
+
+
+> A canister can inspect ingress messages before executing them. When the IC receives an update call from a user, the IC will use the canister method `canister_inspect_message` to determine whether the message shall be accepted.
```

Comment:
`canister_inspect_message` ,- yer mother term?

Reply:
Ah, this is the Wasm-module level interface? Can Motoko actors add such a method? Probably not, since all callables must have a space char in the name.

Reply:
Yes this is literally quoting from the spec so talking about canisters not actors.